### PR TITLE
CASMHMS-6282: Update Alpine base image to resolve CVE

### PR DESCRIPTION
**Pull request by jwlv**
### Summary and Scope

Updated Alpine base image from 3.18 to 3.19 to resolve reported CVE's.  Decided not to use the latest Alpine image 3.21 as I didn't want to inject any more risk than necessary into CSM 1.6.1 just before the end of the release cycle.

Detailed changes include:

- Update Alpine base image to resolve CVE
- New Alpine base image implemented PEP 668 which required switching to a virtual python environment in our dockerfiles
- Broke up RUN directive in dockerfiles for easier reading
- Remove obsolete version field from docker-compose files

Adopted app version 1.36.0 for CSM 1.6.1 (helm chart 3.1.11)

### Issues and Related PRs

* Resolves [CASMHMS-6282](https://jira-pro.it.hpe.com:8443/browse/CASMHMS-6282)

### Testing

Tested on:

* `mug`

Test Description: 

Deployed on mug.  Ran functional tests and confirmed no failures.  Performed various FAS operations and confirmed no issues.  Watched FAS logs to ensure no errors reported.

Testing Check List:

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)? Y
- Were continuous integration tests run? If not, why? Y
- Was upgrade tested? If not, why? Y
- Was downgrade tested? If not, why? Y

## Pull Request Checklist

- [x] Version number(s) incremented, if applicable
- [x] Copyrights updated
- [x] License file intact
- [x] Target branch correct
- [x] CHANGELOG.md updated
- [x] Testing is appropriate and complete, if applicable

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [1.36.0] - 2024-12-18
+
```
Comment (shunr-hpe) [inline CHANGELOG.md:9]: `### Changed`

File: Dockerfile.integration.Dockerfile [hunk 2]
```diff
@@ -47,27 +47,40 @@ RUN set -ex && go build -v -tags musl -o /usr/local/bin/hms-firmware-action gith
...
+    && export LANG="en_US.UTF-8"
+
+#### Python Virtual Environment and Dependencies
+
+RUN python3 -m venv /venv \
+    && . /venv/bin/activate \
+    && pip install --upgrade pip \
+    && pip install pipenv \
+    && pipenv install --deploy --ignore-pipfile
+
```
Comment (shunr-hpe) [inline Dockerfile.integration.Dockerfile:80]: Is the venv still active after the RUN? If so is that intentional?

File: CHANGELOG.md (hunk 1 above)
Comment (jwlv) [reply CHANGELOG.md:9]: Fixed

File: Dockerfile.integration.Dockerfile (hunk 2 above)
Comment (jwlv) [reply Dockerfile.integration.Dockerfile:80]: No, it's not active after the RUN completes.

**Review by mbuchmann-hpe — APPROVED** [review]

File: Dockerfile.integration.Dockerfile (hunk 2 above)
Comment (jwlv) [reply Dockerfile.integration.Dockerfile:80]: All fixed!

**Pull request merged**